### PR TITLE
Remove unused usings

### PR DESCRIPTION
**Pull request by csaba-sagi-sonarsource**

### CODE REVIEW

**Review by mary-georgiou-sonarsource — APPROVED** [review]
LGTM

File: analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs [hunk 1]
```diff
@@ -19,8 +19,6 @@
  */
 
 using System.IO;
-using System.Text;
```
Comment (pavel-mikula-sonarsource) [inline analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs:22]: Why not doing this as part of Mary's PR just to keep the commit history one step smaller? She's changing the file already.

Comment (csaba-sagi-sonarsource) [reply analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs:22]: She asked me to do it separately and yet if the commit history is such a concern, I will ask her to do it as part of her PR.

**Comment by csaba-sagi-sonarsource** [comment]
Will be part of [#6294](https://github.com/SonarSource/sonar-dotnet/pull/6924)

**Pull request closed without merging**